skll/learner/__init__.py

```diff
@@ -125,79 +138,79 @@ class Learner(object):
         size of the learner object in memory and also when
         it is saved to disk.
         Defaults to ``False``.
-    feature_scaling : str, optional
+    feature_scaling : str
         How to scale the features, if at all. Options are
         -  'with_std': scale features using the standard deviation
         -  'with_mean': center features using the mean
         -  'both': do both scaling as well as centering
         -  'none': do neither scaling nor centering
         Defaults to 'none'.
```

Comment:
Just curious... why 'none' as a string here? I think it would be more consistent to have `feature_scaling: Optional[str]` and default to None.

Reply:
The string "none" is the default value for this parameter as we designed it at the time and I don't want to break backwards compatibility at this stage. Probably a good idea for a future release though.